edx_repo_tools/release/tag_release.py

```diff
@@ -773,9 +713,47 @@ def remove_ref_for_repos(repo_names, ref, session, use_tag=True):
     return modified
 
 
-def main():
-    parser = make_parser()
-    args = parser.parse_args()
+@click.command()
+@click.argument(
+    'ref', metavar="REF",
+)
+@click.option(
+    '--tag/--branch', "use_tag", is_flag=True,
+    help="Whether to create branches or tags in the repo"
+)
+@click.option(
+    '--override-ref', metavar="REF",
+    help="A reference to use that overrides the references from the "
+        "repos.yaml file in *ALL* repos. This might be a release candidate "
+        "branch, for example."
+)
+@click.option(
+    '--override', 'overrides',
+    nargs=2, metavar="REPO REF",
+    multiple=True,
+    help="Override a reference for a specific repo. The repo must be "
+         "specified using the full name of the repo, like 'edx/edx-platform'. "
+         "This option can be provided multiple times."
+)
+@click.option(
+    '-y', '--yes', 'interactive', is_flag=True, default=True,
+    help="non-interactive mode: answer yes to all questions"
+)
+@click.option(
+    '-q', '--quiet', is_flag=True, default=False,
+    help="don't print any unnecessary output"
+)
+@click.option(
+    '-R', '--reverse', is_flag=True, default=False,
+    help="delete ref instead of creating it"
+)
+@click.option(
+    '--skip-invalid', is_flag=True, default=False,
+    help="if the repos.yaml file points to an invalid repo, skip it "
+         "instead of throwing an error"
+)
+def main(ref, use_tag, override_ref, overrides, interactive, quiet, reverse, skip_invalid):
```

Comment:
Am I running this wrong? When I try it from your branch, it always prints:
```
/Users/ned/repo-tools/repo-tools/edx_repo_tools/release/tag_release.py:569: Warning: Click detected the use of the unicode_literals __future__ import.  This is heavily discouraged because it can introduce subtle bugs in your code.  You should instead use explicit u"" literals for your unicode strings.  For more information see http://click.pocoo.org/python3/
  @pass_github
```